app/src/main/java/lt/ausra/note/notesActivity/NotesActivityViewModel.kt

```diff
@@ -19,4 +19,18 @@ class NotesActivityViewModel : ViewModel() {
         }
         _notesLiveData.value = NoteRepository.instance.notes
     }
+
+    fun filterNotesList(enteredText: String) {
+        val filteredList = mutableListOf<Note>()
+        NoteRepository.instance.notes.forEach {
```

Comment:
1. I'd consider exporting the filtering function to separate method, easier to distinguish
2. If you're already writing unit tests as exercise, this would be a prime candidate for it
3. Instead of using `forEach`, you may consider using `.filter` instead, which would sort out the items by the boolean and return you a filtered out list.